tests/bluetooth/tester/nrf5340_hci_ipc.conf

```diff
@@ -1,5 +1,6 @@
 # Those have to be the same as in the controller (hci_ipc)
-CONFIG_BT_MAX_CONN=2
+CONFIG_BT_MAX_CONN=3
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=3
```

Comment:
Why do we need multiple advertising sets?

Reply:
This is probably a leftover which will be deleted, but we still need BT_MAX_CONN=3 for tests with three instances of LT

Reply:
That makes sense :)